src/raymath.h

```diff
@@ -2525,4 +2525,54 @@ RMAPI int QuaternionEquals(Quaternion p, Quaternion q)
     return result;
 }
 
+// Decompose a transformation matrix into its rotational, translational and scaling components
+RMAPI void DecomposeMatrix(const Matrix *mat, Vector3 *translation, Quaternion *rotation, Vector3 *scale)
```

Comment:
Note that this function is not following `raymath` conventions (naming, structure), it's going to be reviewed.

Reply:
Sorry I also read the guideline but I missed this one :/